### DISCUSSION TITLE
How do I send API requests from the JS SDK?

### DISCUSSION THREAD

**Discussion by Nv7-GitHub** [pocketbase/pocketbase#532] [Q&A] [answered]
How do I send API requests from the JS SDK so that the authorization header is correct?

**Comment by ganigeorgiev** [comment, marked as answer]
Are you authorizing as admin or user?

In PocketBase they are 2 different things (see https://github.com/pocketbase/pocketbase/discussions/169#discussioncomment-3182726).

You can also find information about authentication in https://pocketbase.io/docs/api-authentication/.

**Reply by Nv7-GitHub** [reply]
User, also I resolved the second issue

**Reply by ganigeorgiev** [reply]
If you want to customize the current authorized token and auth model, you can update the `client.auhtStore` by doing something like:

```js
client.authStore.save(newToken, newUserModel);
```

**Reply by Nv7-GitHub** [reply]
No, I mean that I've extended the framework and added my own API routes, how do I send requests to those with authorization?

**Reply by ganigeorgiev** [reply]
@Nv7-GitHub In this case you could use `client.send(path, config)`, where `config` is the second "options" argument that we pass to `fetch` - https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options.

By default if there is no `config.headers['Authorization']`, it will try to set it for you based on the `client.authStore` state.
